example/server/server.ts

```diff
@@ -32,12 +33,18 @@ export function createServer(): CTProtoServer<AuthorizeMessagePayload, Authorize
 
       throw new Error('Example of unsuccessful auth');
     },
-    async onMessage(message: ApiRequest): Promise<ApiResponse['payload'] | void> {
+    async onMessage(message: ApiRequest | ApiFileRequest): Promise<ApiResponse['payload'] | void> {
       if (message.type == 'sum-of-numbers') {
         return {
           sum: sumNumbers(message.payload),
         };
       }
+      if (message.type == 'file-request') {
+        fs.writeFileSync('./files/'+message.payload.fileName, message.file);
```

Comment:
bad code style (same below)
```suggestion
        fs.writeFileSync('./files/' + message.payload.fileName, message.file);
```